pkg/updater/service/systemd.go

```diff
@@ -68,27 +59,6 @@ func systemdReload() error {
 	return executeCommand(systemdReloadCommand)
 }
 
-func executeCommand(command string) error {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Where is it now?

Reply:
it's here: pkg/updater/service/cmd_executor.go